src/msw/version.rc

```diff
@@ -92,7 +92,11 @@ BEGIN
             VALUE "FileDescription", "wxWidgets " WXLIBDESC " library\0"
             VALUE "FileVersion", wxVERSION_NUM_DOT_STRING "\0"
             VALUE "InternalName", wxSTRINGIZE(WXDLLNAME) "\0"
-            VALUE "LegalCopyright", "Copyright � 1992-2023 wxWidgets development team\0"
+#ifdef _UNICODE
+            VALUE "LegalCopyright", "Copyright " L"\xa9 " "1992-2023 wxWidgets development team\0"
+#else
+            VALUE "LegalCopyright", "Copyright � 1992-2023 wxWidgets development team\0"
```

Comment:
Surely there must be a better way than doing this... I have no idea what exactly is the problem, but we really shouldn't commit this change.

Reply:
Multiple issues here:
1. Windows resource strings are usually charset 1200 anyway (at least as far back as Windows 95, despite its otherwise very limited Unicode support)
2. If `_UNICODE` isn’t defined, the resource charset gets set to ASCII (at line 36), so embedding non-ASCII characters in the text is likely to cause trouble
3. Even in the `_UNICODE` case, using `\xa9` rather than `©` in the source file avoids any confusion over source-file encoding (similar to [the rationale for keeping `languageinfo.cpp` ASCII-only](https://github.com/wxWidgets/wxWidgets/issues/23437#issuecomment-1502938432); see how the browser gets it wrong here in GitHub by assuming it’s UTF-⁠8)
4. Strictly, `_UNICODE` is a Microsoft CRT generic-text thing, not a resource-compiler thing

Unless there’s a particular need ever to define the resource as ASCII-only, I’d remove the existing `#ifdef _UNICODE` stuff at lines 31–​37, always use `04b0` as the resource charset, and encode the copyright sign in the source as `\xa9`.

Reply:
We are use L"\xa9" instead of © symbol in our projects in all case without check. I can't imagine the need to lack unicode.
Code  #ifdef _UNICODE ... #else ... #endif' we include, just so as not to upset the one who represents it :)

It should be borne in mind that always use 04b0 as the resource charset provides for the legal use of © symbol instead of L"\xa9". 
This requires the inclusion  -c65001 option for llvm-rc. Otherwise, we'll get an error 'Non-ASCII 8-bit codepoint (┬) can't be interpreted':
```
[build] llvm-rc: Error in VERSIONINFO statement (ID 1): 
[build] Non-ASCII 8-bit codepoint (┬) can't be interpreted in the current codepage
[build] make[2]: *** [libs/base/CMakeFiles/wxbase.dir/build.make:2061: libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/msw/version.rc.res] Error 1
[build] make[2]: *** Deleting file 'libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/msw/version.rc.res'
```
Of course, trivial solution is use (c) instead of ©, but... :)